lib/spryte/rspec/macros.rb

```diff
@@ -12,7 +12,8 @@ def request(name)
 
         host ENV.fetch("SPRYTE_RSPEC_HOST", "localhost")
 
-        let(:method) { :get }
+        let(:through) { :get }
+        let(:method) { method(:get) }
```

Comment:
Hmm. That makes the specs fail, because `let(:method)` gets called twice. Not sure I want to change the spec to cover this refactoring